forge/config/index.js

```diff
@@ -87,6 +87,15 @@ module.exports = fp(async function (app, opts, next) {
         config.base_url = config.base_url || `http://${config.host}:${config.port}`
         config.api_url = config.api_url || config.base_url
 
+        /** How many days before license expires to notify admins */
+        config.license_warn_period = 30 // TODO: make this configurable
+
+        /** How many days to allow grace after license has expired */
+        config.license_grace_period = 30 // fixed at 30 days
+
+        /** Stops all projects when the grace period is done */
+        config.license_expiry_stops_all = false // If enabled, all projects will be stopped after grace period
+
```

Comment:
I don't think any of these should be user configurable, so they shouldn't be in `forge/config`.

They should be constants in`forge/licensing/index.js`

Reply:
resolved in [7f3c7cc](https://github.com/flowforge/flowforge/pull/1757/commits/7f3c7cc6dd8691259b2b0758d050a5b6abd140ff)